### DISCUSSION TITLE
OpenDTU zeigt 0W anstatt offline und geht darum nicht in Standby

### DISCUSSION THREAD

**Discussion by Amutep** [tbnobody/OpenDTU#1376] [Q&A]
Hallo

Habe einen HM800 über IOBroker mit Python Nulleinspeisungsscript an einem Zendure Solar Flow hängen. Läuft alles ganz gut bis auf das sobald der Akku leer ist am Display der DTU 0W dran steht und die rote Led blinkt. Auf dem Webserver ist das Info- Feld orange und es wird eine Temperatur und Spannung angezeigt. Wahrscheinlich von der letzten Aktualisierung. 
Jetzt die Frage an euch warum wird der WR nicht als offline angezeigt? 

Danke euch

**Comment by DPO99** [comment]
Nunja weil der Wechselrichter ja nicht offline ist. Er hängt ja an der Batterie und wird somit mit Spannung versorgt. Nur weil der Akku "leer" ist, heißt es ja nicht, dass er nicht noch weiter Strom abgeben kann. Ein ausgeschalteter WR braucht vermutlich nicht mehr als 2-3watt.

Offline heißt, dass er aufgrund von zu wenig Spannung/Strom ausgeschalten bzw. wirklich Stromlos wird. Und das passiert nur über Nacht und am Akku eigentlich niemals, außer du trennst den Akku physisch vom WR.

**Comment by huste511** [comment]
Was genau bedeutet denn leer werden?
Ein BMS überwacht und schalten den Akku komplett vom MWR frei?
Oder läuft die Spannung unkontrolliert so weit runter, dass der MWR nicht mehr Einspeisen kann?

An 230V verbrauchen die Hoymiles wirklich 0,0W!
An DC werden erst mal 2-3W weggeschluckt bevor die Einspeisung beginngt.

**Reply by DPO99** [reply]
Na eben nicht, weil wenn das BMS eingreift ist quasi 5 vor 12, deswegen schaltet man den Akku ja vorher ab (20% oder so). Was hast du denn für einen Akku? 

Sofern du den HM nicht physisch über einen Trennschalter oder Sicherung ausschaltest, ist er niemals offline sondern nur ausgeschalten. 

Und da der HM ja nur 2-3w im Leerlauf zieht ist "unkontrolliert" etwas hart formuliert. Je nach Akku Kapazität muss das Ding schon mehrere Tage laufen um "spürbar" was vom Akku wegzufressen.

**Comment by Amutep** [comment]
Danke für eure Antworten. Ich kann nicht genau sagen wie der DC Aus/Eingang vom Zendure Hub bei Akku 0% abgeschaltet wird. 
Aber dann wird vermutlich nur der Strom abgedreht Spannung bleibt weiterhin anliegen ( an der 230V Seite schalt ich eh nix )
Darum zeigt es mir 0W und nicht offline an weil die Spannung noch vorhanden ist. Vermutlich.
